src/handleActions.js

```diff
@@ -3,16 +3,18 @@ import reduceReducers from 'reduce-reducers';
 import invariant from 'invariant';
 import handleAction from './handleAction';
 import ownKeys from './ownKeys';
+import { flattenReducerMap } from './namespaceActions';
 
 export default function handleActions(handlers, defaultState) {
```

Comment:
I'm wondering if this should now take a namespace character like `createActions` does...thoughts?